### PR TITLE
Fix Sphinx errors

### PR DESCRIPTION
**Pull request by vandanavk**
## Description ##
Fix Sphinx errors that appear on local execution of make html.

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- Correct section title in io.py
- Correct prefix usage in conv_rnn_cell.py

## Comments ##
Fixes the following errors:
```
incubator-mxnet/python/mxnet/gluon/contrib/rnn/__init__.py:docstring of mxnet.gluon.contrib.rnn.Conv1DRNNCell:: ERROR: Unknown target name: "conv_rnn".
incubator-mxnet/python/mxnet/gluon/contrib/rnn/__init__.py:docstring of mxnet.gluon.contrib.rnn.Conv2DRNNCell:: ERROR: Unknown target name: "conv_rnn".
incubator-mxnet/python/mxnet/gluon/contrib/rnn/__init__.py:docstring of mxnet.gluon.contrib.rnn.Conv3DRNNCell:: ERROR: Unknown target name: "conv_rnn".
incubator-mxnet/python/mxnet/gluon/contrib/rnn/__init__.py:docstring of mxnet.gluon.contrib.rnn.Conv1DLSTMCell:: ERROR: Unknown target name: "conv_lstm".
incubator-mxnet/python/mxnet/gluon/contrib/rnn/__init__.py:docstring of mxnet.gluon.contrib.rnn.Conv2DLSTMCell:: ERROR: Unknown target name: "conv_lstm".
incubator-mxnet/python/mxnet/gluon/contrib/rnn/__init__.py:docstring of mxnet.gluon.contrib.rnn.Conv3DLSTMCell:: ERROR: Unknown target name: "conv_lstm".
incubator-mxnet/python/mxnet/gluon/contrib/rnn/__init__.py:docstring of mxnet.gluon.contrib.rnn.Conv1DGRUCell:: ERROR: Unknown target name: "conv_gru".
incubator-mxnet/python/mxnet/gluon/contrib/rnn/__init__.py:docstring of mxnet.gluon.contrib.rnn.Conv2DGRUCell:: ERROR: Unknown target name: "conv_gru".
incubator-mxnet/python/mxnet/gluon/contrib/rnn/__init__.py:docstring of mxnet.gluon.contrib.rnn.Conv3DGRUCell:: ERROR: Unknown target name: "conv_gru".
incubator-mxnet/python/mxnet/io/__init__.py:docstring of mxnet.io.NDArrayIter:5: SEVERE: Unexpected section title.

Example usage:
----------
incubator-mxnet/python/mxnet/test_utils.py:docstring of mxnet.test_utils.chi_square_check:3: ERROR: Unexpected indentation.
incubator-mxnet/python/mxnet/test_utils.py:docstring of mxnet.test_utils.chi_square_check:4: WARNING: Block quote ends without a blank line; unexpected unindent.
```

### CODE REVIEW

**Comment by vandanavk** [comment]
thanks @kalyc. the preview for io.html would depend on PR https://github.com/apache/incubator-mxnet/pull/13240. will paste it for the others for now

**Comment by kalyc** [comment]
@mxnet-label-bot add [pr-awaiting-merge]

**Comment by vandanavk** [comment]
Preview:
http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-13252/5/api/python/gluon/contrib.html

http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-13252/5/api/python/io/io.html

http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-13252/5/api/python/tools/test_utils.html

**Comment by vandanavk** [comment]
@nswamy @aaronmarkham for review/merge

**Comment by aaronmarkham** [comment]
@mxnet-label-bot add [Doc]
